pbrEngine/engine.py

```diff
@@ -2012,6 +2057,7 @@ def _distinguishAttack(self, data):
                            slot=slot,
                            moveindex=0,  # FIXME or remove me
                            movename=moveName,
+                           success=self._nextMoveSuccess,
```

Comment:
what is the expected use for this?

Reply:
An external announcer watches the attack callback and plays one line calling out the move if it succeeded, or a different line calling out a miss if it failed.